plutus-core/untyped-plutus-core/test/Evaluation/Machines/Budget/IfThenElse/0.uplc.golden

```diff
@@ -1,47 +1,47 @@
 ( (Right (delay
-            (\z
-              f ->
-               f
+            (\z#0
+              f#1 ->
+               f#1
                  (delay
-                    (\z
```

Comment:
This example demonstrates that without unique suffixes variables `f` and `z` were shadowed multiple times.